lib/interpreters/split_clustering.rb

```diff
@@ -683,7 +683,7 @@ def self.build_partial_service_vrp(service_vrp, partial_service_ids, available_v
       sub_vrp_hash = JSON.parse(sub_vrp.to_json, symbolize_names: true)
 
       vehicle_ids = sub_vrp_hash[:vehicles]&.map{ |v| v[:id] } || []
-      sub_vrp_hash[:services].each{ |service|
+      sub_vrp_hash[:services]&.each{ |service|
```

Comment:
Removing the `&` will force us to remove `allow_blank: false` on services from API input. This doesn't sounds to be a good idea for our will to impose a more clean vrp definition

Reply:
My point was that if we want to use the internal vrps in order to analyse a problem. It requires to edit the vrps manually, or to allow blank for many objects. From my point of view as_json should ease our life both by avoiding marshal dump/load and by making the debug as easy as possible

Reply:
Now I understand your comment, I missed the "if we were to dump the default values" context.

However, what I suggested was to have a `dump_defaults:false/true` flag, so that in the code we have the "full" model with defaults but if we dump for debugging or storage, it is dumped with `dump_defaults:false` which means we don't need to modify the endpoint nor the vrp.

I don't know :/ maybe this is too "heavy" for not much gain. I was just trying to prevent the internal `nil doesn't have x method` errors, it is hard to predict these bugs because it is vrp dependant :/

Reply:
yes !